go/api/pubsub_commands_test.go

```diff
@@ -0,0 +1,156 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import (
+	"fmt"
+	"sort"
+	"time"
+)
+
+func ExampleGlideClient_Publish() {
```

Comment:
a part from another PR?

Reply:
No. Publish command was included in the PubSub Framework PR, but it didn't have examples. Adding it here for completeness.